TalkAloud/TalkAloud/Model/CoreDataManager.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  CoreDataManager.swift
+//  TalkAloud
+//
+//  Created by Justin Bengtson on 6/25/20.
+//  Copyright © 2020 Justin Bengtson. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import CoreData
+
+class CoreDataManager: NSManagedObject {
+    
+    static let sharedInstance = CoreDataManager()
+    
+    
+    func createNewAudioRecording(uniqueFileName: String) -> AudioRecording? {
+        var audioRecordings = [AudioRecording]()
```

Comment:
What's the purpose of this array in this context?

Reply:
I was thinking here that I would store all the AudioRecordings in the AudioRecording array to load later but looking at my code now I already adding content to the array I declare in my `loadAudioRecordings` method.